crates/bevy_ecs/src/system/exclusive_system.rs

```diff
@@ -49,6 +70,8 @@ where
             func: Box::new(self),
             name: core::any::type_name::<F>().into(),
             id: SystemId::new(),
+            // The value of -1 means that everything should be detected as added/changed
```

Comment:
`u32::MAX` is the unsigned representation of `-1`. Even if the implementation is completely unsigned, it conceptually makes more sense to me to name it `-1` since it's being used as "the number just before `0`". I wouldn'd mind changing the comment if you find it confusing.

Reply:
Ah... I agree that it's conceptually -1. Maybe just put it in quotes?

Reply:
Changed in 0406cac5a8c8e4dcb818928e90b7c6d2d8213c23